src/main/java/org/opentripplanner/graph_builder/module/GtfsModule.java

```diff
@@ -155,7 +157,9 @@ public void buildGraph(
     graph.calculateTransitCenter();
   }
 
-  /* Private Methods */
+  public void setDiscardMinTransferTimes(boolean discardMinTransferTimes) {
```

Comment:
I know it' not really a rule but whenever I touched these graph builder modules, I moved the new parameters into the constructor rather than a setter in order to move them towards immutability. What do you think about that?